.NET/Microsoft.Recognizers.Text.Number/NumberRecognizer.cs

```diff
@@ -378,6 +379,20 @@ protected override void InitializeConfiguration()
                     new BaseNumberRangeParser(new HindiNumberRangeParserConfiguration(
                                                   new BaseNumberOptionsConfiguration(Culture.Hindi, options))),
                     new Hindi.NumberRangeExtractor(new BaseNumberOptionsConfiguration(Culture.Hindi, options))));*/
+
+            RegisterModel<NumberModel>(

```

Comment:
Can you add those pass/fail numbers in the description? Models should only be registered when ready-to-ship and we prefer to merge soon and update, if the current coverage of tests is already good.

Reply:
@tellarin, would it be possible to create a small CLI tool that will output those numbers? I had a hard time looking for those numbers during the work on the Dutch language recognizer. Most of the time I was just counting it manually...

(and it would be awesome if you could run the tool and it would mark testcases as supported, if they are supported)

Reply:
This is a great idea and we'll look into adding that. But it may take a while due to other demands.

Reply:
@tellarin - I decided to pull the Ordinal and Percent models from this PR. I will submit the percent model shortly, followed by the ordinal model. I consider this PR be ready for merging.